docs/internal/contributing/README.md

```diff
@@ -59,6 +57,16 @@ Example:
 make CONTAINER_MOUNT_OPTIONS=delegated
 ```
 
+### Running Grafana Mimir locally
+
+An easy way to run Grafana Mimir locally during development is to use the docker-compose setup located at `development/<deployment-mode>/`, e.g.
+
+```bash
+development/mimir-read-write-mode/compose-up.sh -d
```

Comment:
What is `-d` here? I've never used that flag.

Reply:
> 
--detach , -d |   | Detached mode: Run containers in the background
-- | -- | --

see [source](https://docs.docker.com/engine/reference/commandline/compose_up/),

I usually use it because most of the time I'm not interested in the logs of _all_ components, just the ones that have pending changes. Also, I was assuming this is what people use, because we've got the dedicated `compose-down.sh` script to stop containers, otherwise a `Ctrl+C` would do the job of that.

Reply:
ah, thanks! you're right, I always use `ctrl+c` :) good to know about this option. That said, I'm not sure we need to include it in this doc, if default works fine.

Reply:
On the upside, if we do we also get to document the existence/purpose of the tear-down script. :) If you feel strongly about it, I'm happy to remove it.

Reply:
I don't, feel free to merge as-is.